apache-cassandra-mixin/dashboards/cassandra-keyspaces.libsonnet

```diff
@@ -882,7 +896,7 @@ local getMatcher(cfg) = 'job=~"$job", instance=~"$instance", keyspace=~"$keyspac
             template.new(
               'rack',
               promDatasource,
-              'label_values(cassandra_cache_size, rack)',
+              'label_values(cassandra_cache_size{%(cassandraSelector)s, datacenter="$datacenter"}, rack)' % $._config,
```

Comment:
We will want to add the check for the `config.enableDatacenterLabel` value before putting datacenter here.

Reply:
@schmikei i guess you're right. good catch. it would be pretty dumb to enable the rack label but not the datacenter label. but i guess it's possible.

Reply:
updated on all 3 dashboards